### ISSUE TITLE
Send an email notification when a new draft is created

### ISSUE THREAD

**Issue by jonathanthiry** [ICOS-Carbon-Portal/doi#8]

**Comment by mirzov** [comment]
It might be a good option to not send the email directly when the draft DOI is created, but instead when the user presses a dedicated button (that would act kind of like Publish button for admins). We could call the button something like "Submit for publication", and enable it only if the DOI metadata is complete. After pressing, the button should be disabled to prevent accidental spamming. Initially, it should be disabled, too, and enabled only after a successful Update. The email should be sent to all the DOI admin users (we can possibly reduce the number of those now that we have the new workflow in place). @jonathanthiry how does that sound?

**Comment by jonathanthiry** [comment]
Yes, that's much better! I thought it wouldn't work to have a button because we wouldn't know if an email had already been sent or not. However, it will work great if it's only enabled after an update, like you are suggesting. Thanks!
